hack/templates/values/values.template

```diff
@@ -16,6 +16,20 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+admissionController:
+  enabled: false
+  useCertManager: false
```

Comment:
done we want by default:

useCertManager: true
and the entries below to be commented out ?

Reply:
done